.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
cool! Can we make sure the requirement for Python 3.7+ matches what we mention in Readme, Docs etc.